### PR TITLE
Feature/bridge

### PR DESCRIPTION
**Pull request by TinyRogue**

### CODE REVIEW

File: analysis/analysis.py [hunk 1]
```diff
@@ -83,7 +82,7 @@ def get_session_changes_over_time(currency_code: CurrencyCode, time_range: TimeR
     for date, value in american_values.items():
         print(f'At day {date} currency changed come to {value:.2} %')
 
-    print(f'Australian currency rate differences over {TimeRange.LAST_QUARTER.value} days')
+    print(f'Euro currency rate differences over {TimeRange.LAST_QUARTER.value} days')
```
Comment (Roller23) [inline analysis/analysis.py]: It seems to me that AUD was the correct currency, but you have to make sure yourself

**Review by Roller23 — APPROVED** [review]
Other than the currency thing, I think it looks all fine

File: analysis/analysis.py (hunk 1 above)
Comment (TinyRogue) [reply analysis/analysis.py]: You got eyes like a hawk, but I fixed this unimportant part

**Comment by TinyRogue** [comment]
I've added option for specifying `start and end date` for testing purposes and added some examples how to use it _analysis.py_  and _url_builder.py_

**Review by cheretka — APPROVED** [review]
ok

**Pull request merged**